app/components/course-search-result.js

```diff
@@ -0,0 +1,18 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { reads } from '@ember/object/computed';
+
+export default Component.extend({
+  classNames: ['course-search-result', 'results-row'],
```

Comment:
Is results-row still used? Looks like it's gone form the css.

Reply:
Will remove.